docs/details/commands/pmm-admin.md

```diff
@@ -344,18 +362,43 @@ For example, if you want all collectors except `topmetrics`, specify:
 
 ###### Limit dbStats, collStats and indexStats
 
- All the collectors are enabled by default. However, collStats and indexStats are enabled if there are less than 200 collections across all the databases, excluding the admin, config, and local system databases.
+By default PMM decides what limit to set for the number of collections to monitor for the collStats and indexStats collectors. 
```

Comment:
Maybe it's mentioned in another place, but: how will the user know that as for now, by default, these collectors are OFF and --max-collections-limit flag will have no effect until you Enable all collectors?

Reply:
above there is mention `PMM starts the MongoDB exporter by default only with `diagnosticdata` and `replicasetstatus` collectors enabled.`

but we overall maybe need to have some table in the start of the section - what collectors exist, what they do and which are enabled by default.

Reply:
Ok. good to know if this was mentioned. 
This is not a good place to mention what's on what's OFF by default.  

We can add later some Advanced Architecture page  (and migrate our "3 resolution "  page   from the confluence )